.github/workflows/helm-package-release.yaml

```diff
@@ -88,7 +88,7 @@ jobs:
                 git push origin --delete rel-${chart_name}-${chart_version} || true
               fi
               # this command should fail the job if tag already exists
```

Comment:
This

Reply:
Sorry, I did not see your comments in the code. I read it just now. I think we need a discussion on this. Forcing recreation of tag, in any case, should be avoided. Perhaps we just do not allow multiple chart modifications in one PR if the intention is to make a release. Let's talk about this in our 1-1.